### DISCUSSION TITLE
scrolling of the mouse

### DISCUSSION THREAD

**Discussion by faon-92** [SubtitleEdit/subtitleedit#6979] [Ideas]
Is it possible to allow the scrolling of the mouse inside the text box?

**Comment by darnn** [comment]
You can already use the mouse to scroll the text in the text box up and down if you have enough lines (though this probably shouldn't happen under normal circumstances).

**Comment by faon-92** [comment]
ah right.
sometimes I have a long transcript and I need to transforme it to subtitles. so this function can be very useful.
thanks

**Comment by darnn** [comment]
Sure. If you're doing this and don't want to end up with subtitles that are more than two lines, check out the settings in Import plain text (file->import->plain text), particularly auto split. This is what I use:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/19672444/30db5cd4-e286-455c-a79d-7eedf2ca71ef)

And then I further break them up or whatever by hand.

**Comment by faon-92** [comment]
thank you for this nice advice:)

**Comment by faon-92** [comment]
Can I ask you a newbie question, please?

I see how to ajust the dialogue style and constination style, but I don't
know how to create a dialogue in a srt file...
[image: image.png]

thank you very much if you can give me a clue...

On Fri, Jun 2, 2023 at 1:43 PM darnn ***@***.***> wrote:

> Sure. If you're doing this and don't want to end up with subtitles that
> are more than two lines, check out the settings in Import plain text
> (file->import->plain text), particularly auto split. This is what I use:
> [image: image]
> <https://user-images.githubusercontent.com/19672444/242874533-30db5cd4-e286-455c-a79d-7eedf2ca71ef.png>
>
> And then I further break them up or whatever by hand.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SubtitleEdit/subtitleedit/discussions/6979#discussioncomment-6070886>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6MWTDBBFGLNOTO66YAG42DXJHGU7ANCNFSM6AAAAAAYYEOJFM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Feng Sridi

**Comment by darnn** [comment]
Your image didn't upload. If you're replying to this by email, you need to go to the actual GitHub page and reply there, otherwise the attachments are lost. And since I can't see the image, I'm afraid I don't really understand your question.

**Comment by faon-92** [comment]
I got it, and I did create a new discussion in the Q/A session :)

On Fri, Jun 2, 2023 at 4:15 PM darnn ***@***.***> wrote:

> Your image didn't upload. If you're replying to this by email, you need to
> go to the actual GitHub page and reply there, otherwise the attachments are
> lost. And since I can't see the image, I'm afraid I don't really understand
> your question.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SubtitleEdit/subtitleedit/discussions/6979#discussioncomment-6072556>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6MWTDDZJYXZEIM6C2FOBDLXJHYP7ANCNFSM6AAAAAAYYEOJFM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Feng Sridi
